package.json

```diff
@@ -43,7 +43,9 @@
     "fbjs": "^0.8.4",
     "glamor": "^2.15.5",
     "inline-style-prefixer": "^2.0.4",
-    "lodash": "^4.15.0",
+    "is-function": "^1.0.1",
+    "is-plain-object": "^2.0.1",
+    "stylis": "^0.6.1",
```

Comment:
Oh yeah, good catch, my bad! I was trying this over on the #286 branch and then extracted those changes, will remove!